synapse/datamodel.py

```diff
@@ -33,7 +33,10 @@ def getTypeNorm(name, valu):
 def getTypeFrob(name, valu):
     '''
     '''
-    return tlib.reqDataType(name).frob(valu)
+    try:
+        return tlib.reqDataType(name).frob(valu)
+    except:
+        return None,{}
```

Comment:
can we `logger.warn` here if the exception isn't of type `KeyError` or something expected?